CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+**1.1.0 - 04/25/24**
```

Comment:
When this gets approved I will update this date. I do want to confirm that `1.1.0` is the right version number to use.

Reply:
I think this is right